<br>

<!-- <p align="center"><img src="/images/Logo-code.png" width="80"></p> -->
<h1 align="center"> üë©‚Äçüíª <br><br> - Code - </h1>

# public pipelines 

_If you have any question or comment please reach me through the channels listed [there](http://pbaduel.com/about)._

## [SPLITREADER](/SPLITREADER) 

This public repository contains the scripts for running the SPLITREADER pipeline initially developped by [Quadrana et al. eLife 2016](https://doi.org/10.7554/eLife.15716) and further improved by [Baduel et al. MMB 2021](https://doi.org/10.1007/978-1-0716-1134-0_15). 

### [SPLITREADER_v1.0](https://github.com/baduelp/public/tree/master/SPLITREADER/SPLITREADER_v1.0)

This folder contains the scripts to detect non-reference transposable element insertions from short-read sequencing data as described in [Baduel et al. MMB 2021](https://doi.org/10.1007/978-1-0716-1134-0_15). <br/>
Accessory files and wrapper scripts are also available in [/thaliana](https://github.com/baduelp/public/tree/master/SPLITREADER/SPLITREADER_v1.0/thaliana) to run the SPLITREADER pipeline on the _A. thaliana_ genome. <br/>

### [SPLITREADER_v1.2](https://github.com/baduelp/public/tree/master/SPLITREADER/SPLITREADER_v1.2)

This folder contains a snakemake version of the SPLITREADER pipeline produced by [@aurelpetit](https://github.com/aurelpetit) for easier portability and management of dependencies. 

## For citation

**Baduel P**, Quadrana L, Colot V. Efficient detection of transposable element insertion polymorphisms between genomes using short-read sequencing data. Plant Transposable Elements, Methods in Molecular Biology, [10.1007/978-1-0716-1134-0_15](https://doi.org/10.1007/978-1-0716-1134-0_15), 04/2021.




<!---
## SPLITREADER pipeline 
The scripts to detect non-reference transposable element insertions from short-read sequencing data, as described in Baduel et al. MMB [2021](https://doi.org/10.1007/978-1-0716-1134-0_15), are available on my public GitHub [repository](https://github.com/baduelp/public/tree/master/SPLITREADER). 
<br>
<br>
In addition, accessory files and wrapper scripts are supplied to run the SPLITREADER pipeline on the _A. thaliana_ genome. <br/> 

## MATLAB genomic toolbox
Custom MATLAB functions for genomics can be downloaded from [here](https://github.com/baduelp/public/tree/master/MATLAB). 

### Circular distribution across chromosomes (Circos-plots)

Upcoming soon

### Map of markers with color-coded values

<p align="center">
<img src="/images/Position of populations.png" >
</p>

`plot_POPmap_function.m`
  
  This Matlab function displays markers on a world Mercator-projection map with color-coded values to display quantitative or categorical information. 

  

### Principal Component Analysis (PCA) of Allele-Frequency population data

<p align="center">
<img src="/images/PC1 & 2 of fake PCA centered 23-Jun-2017.png" style="margin-right: 15px;" width="400">
</p>

`plot_AF_PCA.m` 
  
  This Matlab function performs a principal component analysis of allele-frequency population data from next-generation sequencing. It takes as input the N x M matrix containing for each of N sites the allele-frequency of derived alleles in each of the M individuals/populations.
 --->

        
