# Research <img align="right" src="/images/seedling.png" style="margin-right: 0px;" width="80">

<p align="justify">
My research focuses on the evolution of plant genomes to understand the genetic mechanisms involved and their consequences for adaptation and evolution. I am particularly interested in the evolutionary consequences of <b>transposable element</b> (TE) dynamics and <b>polyploidy</b>, which are both major drivers of genome evolution.
</p>

### Assessing how the genetic and environmental determinants of natural transposition shape its adaptive potential in _Arabidopsis thaliana_

<p align="justify">
Standing genetic variation is generally thought to be the main source of rapid adaptation to environmental changes. As a result, genomic studies aimed at estimating the evolutionary potential of native populations in future climates have mostly focused on SNPs. However, there is increasing evidence that the rare and typically large effect alleles created by TE insertions, even though mostly deleterious, can be sometimes beneficial and contribute significantly to local adaption. Leveraging the multi-omic and bio-climatic data for more than 1,000 wild <em>A. thaliana</em> accessions sequenced as part of the 1001 Genomes project, we set out to determine the rate of TE mobilization and its potential to create adaptive variation in natural settings. 
</p>

<img align="center" src="/images/GBIO_summary-fig.png" >

<p align="justify">
Using an updated pipeline to detect TE insertion polymorphisms (TIPs) from short-read sequencing data, we identified >23,000 TIPs across >1000 accessions from across the globe. Based on this dataset, we determined the substitution rate of TIPs in nature to be almost a third of that of SNPs (between 0.06-0.08 TE insertion per genome per generation) and very close to the actual transposition rate we measured experimentally. However, unlike SNPs, TE insertions tend to have large deleterious effects when they insert within or near genes (which occurs at least 50% of the time for some TE families) and are therefore rapidly purged by purifying selection. Nonetheless, we found some gene loci to be recurrently targeted by TE insertions (most notably <em>FLC</em>’s 1st intron), and these tended to be found in disturbed environments where they showed signatures of positive selection.
</p>

<p align="justify">
Furthermore, we also found that a naturally truncated variant of <em>NRPE1</em> (a key component of the RdDM pathway which targets DNA methylation over TEs) causes lower CHH methylation and higher levels of transposition, largely by enhancing the environmental sensitivity (GxE) of TEs. Strikingly, this mutator <em>NRPE1</em> allele is enriched at the edge of the environmental niche of <em>A. thaliana</em> where it itself shows signature of positive selection, akin in a way to bacterial mutator alleles that can be temporarily beneficial in harsh environments. How such an allele has been retained over long term in <em>A. thaliana</em> opens exciting avenues of inquiry, especially in times of climate crisis where TEs may be essential genomic players in the demise or rescue of native populations.
</p>


**Baduel P**, Leduque B, Ignace A, Gy I, Gil J, Loudet O, Colot V, Quadrana L. _Genetic and environmental modulation of transposition shapes the evolutionary potential of Arabidopsis thaliana_. Genome Biology, [10.1186/s13059-021-02348-5](https://doi.org/10.1186/s13059-021-02348-5), 05/2021. 

### Evaluating transposable elements dynamics following auto-polyploidization in _Arabidopsis arenosa_

<img align="left" src="/images/DSC01221.JPG" style="margin-right: 15px;" width="175">

<p align="justify">
During my PhD with Professor Bomblies at Harvard University, followed by a short postdoc at the John Innes Centre in Norwich, UK, I investigated the impact of whole-genome duplication (WGD) on genome evolution in the young natural diploid / autotetraploid system offered by <em>A. arenosa</em>. Leveraging a large genome resequencing dataset encompassing 300 samples of both diploid and tetraploid <em>A. arenosa</em>, I showed that polyploids present increased non-synonymous sequence diversity, and established it was due mainly to increased effective population sizes with an additional effect from the genetic masking of deleterious mutations associated with polysomy. 
</p>

<p align="justify">
In the group of Dr. Colot at the IBENS in Paris, I have initiated a collaborative project to further investigate the genome evolution of autopolyploids in relation to transposable elements (TEs) dynamics. Due to their mostly deleterious effects, TE insertions are indeed predicted to benefit significantly from the genetic masking I showed was at play in autotetraploids. In addition, transposition bursts have been proposed to follow the genome shock of auto-polyploidization, and both mechanisms have been hypothesized to underlie the increased TE content typically observed in polyploids. Using the bioinformatics tools developed by the Colot group to analyze the genomic sequences available in <em>A. arenosa</em>, I was able to comprehensively measure TE mobilization in both diploids and tetraploids. Thus, I showed that the increased TE content observed in tetraploid <em>A. arenosa</em> was mostly driven by the genetic masking associated with polyploidy. In contrast, I could not find any evidence of a transposition burst dating back to the ancestral polyploidization event, demonstrating that WGD is not sufficient to induce such a response and may require additional genetic or environmental determinants. 
</p>

<img align="center" src="/images/NatComm2019_Figure 5.png" >

<p align="justify">
Nonetheless, I find evidence that TE insertions can be more than just a burden and occasionally contribute to the adaptive potential of polyploid populations. In tetraploid <em>A. arenosa</em>, TE insertions appear to be under local positive selection when within environmentally responsive genes notably the major flowering-time repressor gene <em>FLOWERING LOCUS C</em> (<em>FLC</em>). At this locus, I identified an exonic TE insertion that is specifically associated with the rapid-cycling tetraploid lineage that colonized railways, supporting the idea that TEs can generate alleles enabling rapid adaptation to novel habitats. Yet, the higher TE load in polyploids suggests that in the long run, the accumulation of deleterious mutations, in particular genic TE insertions, could have a major negative impact on the evolution of polyploids. This is one of the main reasons why I argued that, unless selected for, polyploidy is a transient state that will likely eventually result in extinction, or stabilization of the lineage by re-diploidization. I envision something we called a ‘polyploid hop’, in which the increased genetic diversity, as well as other mechanistic and physiological advantages of polyploid populations, may enable adaptation to new and more challenging environments. In the process, TEs might potentially also reshape genomes by causing ectopic recombination, which could lead to genome divergence and reproductive isolation.
</p>

**Baduel P**, Quadrana L, Hunter B, Bomblies K, Colot V. _Relaxed purifying selection in autopolyploids drives transposable element over-accumulation which provides variants for local adaptation_. Nature Communications,[10.1038/s41467-019-13730-0](https://doi.org/10.1038/s41467-019-13730-0), 12/2019

Monnahan P\*, Kolář F\*, **Baduel P**\*, Sailer C, Koch J, Horvath R, Laenen B, Schmickl R, Paajanen P, Fuxová G, Holcová M, Arnold B, Weismann C, Marhold K, Slotte T, Bomblies K, Yant L. _Pervasive population genomic consequences of genome duplication in_ A. arenosa. Nature Ecology & Evolution, [10.1038/s41559-019-0807-4](https://doi.org/10.1038/s41559-019-0807-4), 02/2019  
*_authors contributed equally_



